specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json

```diff
@@ -2258,6 +2230,33 @@
         }
       },
       "description": "Response for the ListExpressRouteServiceProvider API service call."
+    },
+    "CircuitConnectionStatus": {
+      "type": "string",
```

Comment:
The "readOnly" field should be placed here as every schema that references this definitions contains it.